com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using Unity.Barracuda;
+using Unity.MLAgents.Actuators;
+using Unity.MLAgents.Inference;
+using Unity.MLAgents.Policies;
+using Unity.MLAgents.Sensors;
+using UnityEditor;
+using UnityEditor.Analytics;
+using UnityEngine.Analytics;
+
+namespace Unity.MLAgents.Analytics
+{
+    internal class InferenceAnalytics
+    {
+        static bool s_EventRegistered = false;
+        const int k_MaxEventsPerHour = 1000;
```

Comment:
From the code, it looks like there will ever only be one event sent for "InferenceModelSet". Can you add some more comments on these const so we know what they do and how they are used?

Reply:
The values were copied from the example code. We'll eventually have more event types to send; we can generalize then.

As for what they do: https://docs.unity3d.com/ScriptReference/EditorAnalytics.RegisterEventWithLimit.htm

Reply:
Added a few comments (mostly from the docstrings from `RegisterEventWithLimit()`)